src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex

```diff
@@ -0,0 +1,1045 @@
+\label{sec:model:InterfaceTemplateListRequest}
+ 
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3cm} | p{5cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+authentication & \hyperref[sec:model:Identity]{Identity} & yes & The requester of the operation. \\ \hline
+interfaceTemplates & \pref{List}$<$\hyperref[sec:model:InterfaceTemplateRequest]{InterfaceTemplateRequest}$>$ & yes & A list of interface templates. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{InterfaceTemplateRequest}
+\label{sec:model:InterfaceTemplateRequest}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3.8cm} | p{6.3cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+name & \pref{Name} & yes & Unique name of the interface template. \\ \hline
+protocol & \pref{Protocol} & yes & Protocol of the interface template. \\ \hline
+propertyRequirements &  \pref{List}$<$\hyperref[sec:model:InterfaceTemplatePropertyRequest]{InterfaceTemplatePropertyRequest}$>$ & yes & Properties of the interface template. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{InterfaceTemplatePropertyRequest}
+\label{sec:model:InterfaceTemplatePropertyRequest}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3.8cm} | p{4cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+
+name & \pref{String} & yes & Name of the property. \\ \hline
+mandatory & \pref{Boolean} & yes & True if the property is mandatory, false if optional. \\ \hline
+validator & \pref{PropertyValidator} & no & Name of the validator assigned to the property. \\ \hline
+validatorParams & \pref{List}$<$\pref{String}$>$ & no & Parameter values of the validator. \\ \hline
+\end{tabularx}
+\end{table}
+
+\clearpage
+
+\msubsection{struct}{InterfaceTemplateRemoveRequest}
+\label{sec:model:InterfaceTemplateRemoveRequest}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{4.1cm} | p{4.8cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+authentication & \hyperref[sec:model:Identity]{Identity} & yes & The requester of the operation. \\ \hline
+names &  \pref{List}$<$\pref{InterfaceTemplate}$>$ & yes & Names of the interface templates that need to remove. \\ \hline
+\end{tabularx}
+\end{table}
+
+\subsection{Primitives}
+\label{sec:model:primitives}
+
+Types and structures mentioned throughout this document that are assumed to be available to implementations of this service.
+The concrete interpretations of each of these types and structures must be provided by any IDD document claiming to implement this service.
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3cm} | X |} \hline
+\rowcolor{gray!33} Type & Description \\ \hline
+\pdef{Address}          & A string representation of the address. \\ \hline
+\pdef{AddressType}      & Any suitable type chosen by the implementor or service. \\ \hline
```

Comment:
I think you meant implementor **of the** service (or **of** service)

Reply:
I Also had to fix it in some previous documents.